tests/TestCase/Controller/Component/RequestHandlerComponentTest.php

```diff
@@ -336,7 +337,7 @@ public function testViewClassMap()
     public function testIsAjaxParams()
     {
         $this->request->env('HTTP_X_REQUESTED_WITH', 'XMLHttpRequest');
-        $event = new Event('Controller.startup', $this->Controller);
+        $event = new ControllerEvent('Controller.startup', $this->Controller);
```

Comment:
This has now been implemented as can be seen in: https://github.com/cakephp/cakephp/pull/7173/files#diff-d2431a37a863974d64b7122b7d288d9aR35